Apps/Examples/Examples/All Examples/SwiftUIExample.swift

```diff
@@ -137,7 +137,7 @@ internal class SwiftUIMapViewCoordinator {
         _camera = camera
     }
 
-    func notify(for event: Event) {
+    func notify(for event: MapboxCoreMaps.Event) {
```

Comment:
I think we should add a couple of type aliases:

```swift
typealias Event = MapboxCoreMaps.Event
typealias TelemetryEvent = MapboxCommon.Event
```

so that it's nicer at the point of use for developers.

thoughts?